frontend/src/actions/resources.js

```diff
@@ -2,6 +2,10 @@ import { sicknessStatus, testStatus } from "routes/types";
 
 export const getStoryResources = (story) => {
   const resources = [
+    {
+      text: "All available resources",
+      site: "/resources",
```

Comment:
This does not work .

`Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
`

Reply:
Deleted